hack/deploy-sriov-in-telco-ci.sh

```diff
@@ -105,6 +107,7 @@ if [ -n "$PR" ]; then
   echo "cloning sriov repo from PR ${PR}"
   git fetch origin pull/$PR/head:pr
   git checkout pr
+  git rebase $BRANCH
```

Comment:
I'm not 100% sure about this. It may generate a conflict, which we'd discover after the cluster provisioning (~1h). Does it solve any specific case?

Reply:
I think in a CI system we must always rebase on the branch we want to merge to. this way we can catch regressions created by the combination of two or more separate PRs getting merge